docs/tutorial/kymographs.rst

```diff
@@ -75,116 +105,148 @@ There are also several properties available for convenient access to the kymogra
 
-    kymo_sliced = kymo["114.2s":"164.6s"]
-    kymo_cropped = kymo_sliced.crop_by_distance(6, 24)
+    .. image:: figures/kymographs/kymo_cropped_and_sliced.png
 
-    kymo_cropped.plot("green")
+Finally, we can also flip a kymograph along its positional axis using :meth:`~lumicks.pylake.kymo.Kymo.flip()`.
+This returns a new (but flipped) :class:`~lumicks.pylake.kymo.Kymo`::
 
-.. image:: figures/kymographs/kymo_cropped_and_sliced.png
+    kymo_flipped = kymo.flip()
+
+    plt.figure()
+    plt.subplot(211)
+    kymo.plot("rgb", adjustment=lk.ColorAdjustment(0, 98, mode="percentile"))
+
+    plt.subplot(212)
+    kymo_flipped.plot("rgb", adjustment=lk.ColorAdjustment(0, 98, mode="percentile"))
+
+    plt.tight_layout()
+    plt.show()
+
+.. image:: figures/kymographs/kymo_flipped.png
 
 Calibrating to base pairs
 -------------------------
 
-By default, kymographs are constructed with units of microns for the position axis. If, however, the kymograph spans a known length of DNA (for example,
-lambda DNA) we can calibrate the position axis to kilobase pairs::
+By default, kymographs are constructed with units of microns for the position axis. If, however, the kymograph spans a known length of DNA
+(here for example, lambda DNA) we can calibrate the position axis to kilobase pairs (kbp)::
 
     kymo_kbp = kymo_cropped.calibrate_to_kbp(48.502)
 
 Now if we plot the image, the y-axis will be labeled in kbp::
 
+    plt.figure()
     kymo_kbp.plot("green")
+    plt.show()
 
 .. image:: figures/kymographs/kymo_calibrated.png
 
-These units are also carried forward to any downstream operations such as
-kymotracking algorithms and MSD analysis, . *Note: currently this is a static calibration, meaning it is only valid
-if the traps do not change position during the time of the kymograph.*
+These units are also carried forward to any downstream operations such as kymotracking algorithms and MSD analysis.
 
-We can also interactively slice, crop, and calibrate kymographs using::
+.. warning::
 
-    widget = kymo.crop_and_calibrate(channel="green", tether_length_kbp=48.502)
-    plt.show()
+    Currently this is a static calibration, meaning it is only valid if the traps do not change position during the time of the kymograph.
+
+    Also, the accuracy of the calibration is dependent on how the kymo is cropped. If you crop the kymo by visually estimating the
+    bead edges, the resulting position should be taken as approximate.
+
+We can also interactively slice, crop, and calibrate kymographs using :meth:`~lumicks.pylake.kymo.Kymo.crop_and_calibrate`::
+
+    widget = kymo.crop_and_calibrate(channel="rgb", tether_length_kbp=48.502, aspect="auto", adjustment=lk.ColorAdjustment(0, 99.5, mode="percentile"))
 
 .. image:: figures/kymographs/kymo_interactive.png
 
 Simply click and drag the rectangle selector to the desired ROI. After closing the widget, we can access the edited kymograph
```

Comment:
You didn't edit this, but what does closing the widget mean in a lab context?

Reply:
yeah, when I originally wrote this and tested the plot I wasn't using `jupyter` at all (surprising, I know) so you have to close the window before the code to grab the new kymo is executed. I just removed the whole sentence